app/classifier/drawing-tools/freehand-helpers.coffee

```diff
@@ -0,0 +1,8 @@
+createPathFromCoords = (coordsArray) ->
+  [firstCoord, otherCoords...] = coordsArray
+  path = "M #{firstCoord.x},#{firstCoord.y} "
+  path += "L #{x},#{y} " for {x, y} in otherCoords
```

Comment:
When a path is closed by the drawing tools, it adds the first point back to the end of the coords array. Reason being, while we use SVG for paths on PFE, researchers might not, and this is probably the clearest way of showing it's a closed shape.